world/test_chargen.py

```diff
@@ -15,11 +15,11 @@
 class ChargenTestCase(EvenniaTest):
     """Test case for the chargen menu process."""
     character_typeclass = Character
-    settings.PROTOTYPE_MODULES = PROTOTYPE_MODULES
 
     def setUp(self):
         super(ChargenTestCase, self).setUp()
         self.session.msg = Mock()
+        settings.PROTOTYPE_MODULES = PROTOTYPE_MODULES
```

Comment:
This may not be necessary any longer, as I believe this was a workaround for using the game's settings file before the `evennia test --settings settings.py gamedir` syntax was introduced.